### ISSUE TITLE
Display title of empty feeds

### ISSUE THREAD

**Issue by h43z** [newsboat/newsboat#182]
**Newsboat version (copy from `newsboat -v`)**:
2.11.1
Steps to reproduce the issue:
Use any feed that has no items yet.
```
<?xml version="1.0" encoding="UTF-8"?><rss xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:atom="http://www.w3.org/2005/Atom" version="2.0"><channel><title><![CDATA[test title]]></title><description><![CDATA[test description]]></description><link>http://test.com</link><generator>test</generator><lastBuildDate>Sat, 05 May 2018 23:26:26 GMT</lastBuildDate></channel></rss>
```

**Other info you think is relevant**:
I provide a dynamic rss feed which at the time of importing to newsboat could have NO items. 
It would be nice to display the title of the feed. Right now it just shows the url (or file path) as long as there is not at least one item.

**Comment by Minoru** [comment]
Reproduced with current HEAD, b59ef97d15c913ab51d80e78fe41826965f2c41d.

The culprit is, obviously, [this line](https://github.com/newsboat/newsboat/blob/b59ef97d15c913ab51d80e78fe41826965f2c41d/src/controller.cpp#L913), and the question is: why is it even there? Need to dig through the history to find out before we remove it.

Thanks for an easily reproducible bug report, @h43z!

**Comment by dennisschagt** [comment]
Looks like #732 is a duplicate of this issue, which got resolved a few days ago (in #746).
